projects/schematics/src/shared/constants.ts

```diff
@@ -1081,6 +1081,10 @@ export const CLI_CART_SAVED_CART_FEATURE = 'Saved-Cart';
 
 export const CLI_CART_QUICK_ORDER_FEATURE = 'Quick-Order';
 
+export const CLI_CART_MAIN_FEATURE = 'Cart';
```

Comment:
can we call it `main cart` instead of `cart`?

Reply:
Has to match the value passed by the installer menu